bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -1,17 +1,17 @@
-import React, { PureComponent } from 'react';
+import React, { } from 'react';
```

Comment:
Just out of curiosity, why sometimes you use named import and sometimes you use `React.`?